frontend/src/views/web/change-password.html

```diff
@@ -7,7 +7,7 @@
                     <form name="changepwdForm" ng-submit="changepwd.changePassword(changepwdForm.$valid)" novalidate>
 
                         <div class="input-field align-left">
-                            <input id="old_password" name="old_password" type="password" class="validate text-dark-black dark-autofill w-400" ng-model="changepwd.user.old_password"  ng-minlength="8" required focus-if>
+                            <input id="old_password" name="old_password" type="password" text-dark-black dark-autofill w-400" ng-model="changepwd.user.old_password"  ng-minlength="8" required focus-if>
```

Comment:
You had not to delete `class`.

before: `class="validate text-dark-black dark-autofill w-400"`
after : `class="text-dark-black dark-autofill w-400"`